pkg/capabilities/consensus/ocr3/types/ocr3_types.proto

```diff
@@ -15,5 +15,24 @@ message AggregationOutcome {
 
 message Query {
   // the requests to get consensus on.
-  repeated string requestIds = 1;
+  repeated RequestId requestIds = 1;
+}
+
+message RequestId {
+  string requestId = 1;
+  string namespaceId = 2;
+}
+
+message Request {
```

Comment:
This name "Request" is a little confusing. How about "SingleObservation"? Or "ObservationWithInfo"?